controllers/user.js

```diff
@@ -220,10 +221,17 @@ const setRolePermissions = function(req, role) {
     grantWanAdvancedInfo: req.body['grant-wan-advanced-info'],
     grantVoipInfo: isNaN(voipGrant) || voipGrant < 0 || voipGrant > 2 ?
       0 : voipGrant,
+    grantFlashboardResume: req.body['grant-flashboard-resume'],
   };
 
   let audit = {};
   if (!role) {
+    // Collapsing this field to false since this can be not present in the
+    // request body
+    req.body['grant-flashboard-resume'] =
+      (req.body['grant-flashboard-resume'] === undefined)
+        ? req.body['grant-flashboard-resume'] : false;
```

Comment:
Isso aqui não vai atualizar o `roleParams` que vai ser usado pra salvar na base, e o parse de req.body já foi feito lá em cima. Teria que operar direto em `roleParams`:
```suggestion
    if (roleParams.grantFlashboardResume === undefined) {
      roleParams.grantFlashboardResume = false;
    }
```